### ISSUE TITLE
RF Tools DIM tweaks

### ISSUE THREAD

**Issue by JasonMcRay** [JasonMcRay/InfiTech-Reloaded#4]
Power costs, rarity and bans

**Comment by TedKnudsen** [comment]
Jason

The dims themselves don't look absurdly OP so far.  We don't have any at all for GT-specific ores as far as I can tell, and any fuels seem to cost tens of thousands of RF/t just to run that dimension.

Just preliminary thoughts.  There's a strong likelihood there's some exploits I haven't seen yet.

**Comment by JasonMcRay** [comment]
The comment I posted basically means: Make RFTools usable and find possible issues/exploits.

Dimlet bans will probably not happen, or they will but only in really low ammount. Power costs will be bit tweaked, because I want RFTools dimensions to really be end-game for all of those players who have Fusion Reactor and have no idea what to do with all that energy :D

**Comment by pyure** [comment]
Makes sense.  And I'm not sure why I keep accidentally posting things here from my work github account, sorry about the confusion :\  (pyure=TedKnudsen)

**Comment by JasonMcRay** [comment]
No confusion for me... I already knew that TedKnudsen and Pyure is one and same person :)
